qiskit/circuit/parameter.py

```diff
@@ -114,6 +114,16 @@ def name(self):
         """Returns the name of the :class:`Parameter`."""
         return self._name
 
+    @property
+    def uuid(self) -> UUID:
+        """Returns the UUID of the :class:`Parameter`.
```

Comment:
We do have intersphinx setup with python so I think you can do:

```suggestion
        """Returns the :class:`.UUID` of the :class:`Parameter`.
```

and it'll point to https://docs.python.org/3/library/uuid.html#uuid.UUID

Unless the type hint will resolve to that via sphinx magic already, then this isn't needed.

Reply:
I checked and didn't see the type hint resulting in a link. The type hint for the property doesn't seem to be reported at all. It looks like support was [added in Sphinx 4.0](https://github.com/sphinx-doc/sphinx/pull/8983). Maybe Qiskit's custom Sphinx formatting interferes? The properties show up labeled as "Attributes".

Reply:
you can't use `.` suffix matches with intersphinx - it needs to be a fully specified name.  In this case, `` :class:`~uuid.UUID` `` should do what you want.